apps/dataflow/src/scraper/fms/upload_svg_to_s3.py

```
@@ -0,0 +1,46 @@
+LOCAL_SVG_FOLDER = "floorplan_svg"
+# S3 destination path
+S3_DESTINATION = "floorplan_svg"
+
+
+def upload_all_svg_files():
+    """Upload all SVG files from floorplan_svg folder to S3, preserving folder structure"""
+    success_count = 0
+    fail_count = 0
+
+    # Iterate through building folders
+    for building in os.listdir(LOCAL_SVG_FOLDER):
```

Ruff (0.14.14)

[warning] Use `pathlib.Path.iterdir()` instead.

(PTH208)